package.json

```diff
@@ -51,7 +51,6 @@
     "passport": "^0.6.0",
     "passport-google-oauth20": "^2.0.0",
     "prettier": "^3.0.3",
-    "server": "file:",
```

Comment:
Ive removed this dependency because i havent heard objections and it looks like some leftover that didnt get removed. it was added in this commit https://github.com/Educado-App/educado-backend/pull/158/commits/061cc61028926705268c1bac1ccba4ebd343293a @Laskiri safe to remove? 😄